src/main/file.ts

```diff
@@ -0,0 +1,73 @@
+import path from 'path'
```

Comment:
This new `./src/main` directory is meant to hold util files related to the main process. There's more files that need to be moved here eventually.